test/clt-tests/migration-es-ms/index_data.json

```diff
@@ -0,0 +1,464 @@
+{"_index":"test-index","_id":"YYbCv44BmW3FuElXmIAG","_score":1,"_source":{"log":{"level":"INFO"},"field17":"\"elasticsearch.cluster.name\"","field18":"\"docker-cluster\"","message":"2024-04-08 23:48:05 {\"@timestamp\":\"2024-04-08T21:48:05.731Z\", \"log.level\": \"INFO\", \"message\":\"starting ...\", \"ecs.version\": \"1.2.0\",\"service.name\":\"ES_ECS\",\"event.dataset\":\"elasticsearch.server\",\"process.thread.name\":\"main\",\"log.logger\":\"org.elasticsearch.node.Node\",\"elasticsearch.node.name\":\"7b293a37e356\",\"elasticsearch.cluster.name\":\"docker-cluster\"}","field11":"\"process.thread.name\"","field12":"\"main\"","field10":"\"elasticsearch.server\"","@timestamp":"2024-04-08T23:48:05.000+02:00","field":"\":\"","extra_timestamp":"2024-04-08T21:48:05.731Z","field15":"\"elasticsearch.node.name\"","field16":"\"7b293a37e356\"","field13":"\"log.logger\"","field14":"\"org.elasticsearch.node.Node\"","field7":"\"service.name\"","field6":"\"1.2.0\"","field9":"\"event.dataset\"","field8":"\"ES_ECS\"","field3":"\"message\"","field2":"\"log.level\"","field5":"\"ecs.version\"","field4":"\"starting ...\""}}
```

Comment:
Let's put both these files `index_data.json` and `index_mapping.json` to archive and unarchive them directly during test running

Reply:
done